src/Propulsion.Cosmos/PropulsionCosmosPrometheus.fs

```diff
@@ -4,111 +4,155 @@ namespace Propulsion.CosmosStore.Prometheus
 namespace Propulsion.Cosmos.Prometheus
 #endif
 
+(*
+    This file implements a Serilog Sink `LogSink`
+    which publishes metric values to Prometheus.
+
+    It takes in an additional set of custom tags
+    to annotate the metric we're publishing.
+
+*)
+
+type TagNames = string array
+type TagValues = string array
+type Tags = TagNames * TagValues
```

Comment:
friends don't  let friends do type aliases
... only half kidding...
There are very few type aliases in Equinox or Propulsion and I'd like to keep it that way
1. I just spent a minute figuring out what this was
2. Just because Ionide renders it nicely, does not mean github, Rider or vim does
3. I dont think it happens to objectively make it clearer in this instance

So... can we do it the same as the origin PR please?